app/models/circumstanceInfo/CircumstanceDetails.scala

```diff
@@ -16,31 +16,35 @@
 
```

Comment:
What do we need CustomerDetails for?

Reply:
The only place is displaying client name on the confirmation page for agents